README.md

```diff
@@ -1,5 +1,15 @@
+[![Stars](https://img.shields.io/github/stars/laminlabs/bionty?logo=GitHub&color=yellow)](https://github.com/laminlabs/bionty)
+[![pypi](https://img.shields.io/pypi/v/bionty?color=blue&label=pypi%20package)](https://pypi.org/project/bionty)
+
 # bionty: Basic biological entities
 
 bionty is documented & tested within lamindb: github.com/laminlabs/lamindb.
 
+- Access public ontologies of biological entities
+- Manage synonyms, abbreviations, and hierarchical relationships
+
+## Documentation
+
 See [docs](https://lamin.ai/docs/bionty).
+
+More details on concepts, a comparison to other tools, and extending standalone `Bionty base` can be found in the [docs/guide](docs/guide) folder.
```

Comment:
I don't really like the idea of these "2nd-class" docs or even referring to "bionty base" as a standalone thing. I'm not against people using it standalone, and I think this can be a note somewhere in the rendered docs, but here on the readme it looks somewhat half-cooked.

I'd remove this sentence.